css-rules.js

```diff
@@ -8,81 +8,72 @@ module.exports = {
   rules: {
     // GDS predominantly uses a conventional to specify opacity values with
     // decimals
-    // https://stylelint.io/user-guide/rules/list/alpha-value-notation/
+    // https://stylelint.io/user-guide/rules/alpha-value-notation/
```

Comment:
Nice one fixing these links!

Reply:
Thanks @kevindew. Another one that made sense in https://github.com/alphagov/stylelint-config-gds/pull/44/commits/ff52d2586009e9caa2f0a22235ee3349c3c9c84b via #44